client/additional-methods-setup/methods-selector/add-payment-methods-task.js

```diff
@@ -194,7 +198,9 @@ const AddPaymentMethodsTask = () => {
 								<PaymentMethodCheckbox
 									checked={ paymentMethodsState.card }
 									onChange={ handlePaymentMethodChange }
-									fees="missing fees"
+									fees={ formatMethodFeesDescription(
```

Comment:
I'm removing the `fees` prop from `PaymentMethodCheckbox` and having `PaymentMethodCheckbox` consume `WCPaySettingsContext` directly, so that we can just pass the method's name and the component knows how to render the fees.